realm/realm-library/src/main/cpp/CMakeLists.txt

```diff
@@ -204,7 +204,7 @@ endif()
 set(WARNING_CXX_FLAGS "-Werror -Wall -Wextra -pedantic -Wmissing-declarations \
     -Wempty-body -Wparentheses -Wunknown-pragmas -Wunreachable-code \
     -Wno-missing-field-initializers -Wno-unevaluated-expression -Wno-unreachable-code \
-    -Wno-c99-extensions")
+    -Wno-c99-extensions -Wno-dtor-name")
```

Comment:
why adding this flag?

Reply:
Due to https://github.com/realm/realm-core/issues/4732#issuecomment-854473573